packages/material-ui/src/NativeSelect/index.js

```diff
@@ -1 +1,3 @@
 export { default } from './NativeSelect';
+export { default as nativeSelectClasses } from './nativeSelectClasses';
```

Comment:
Convention with the other files

```suggestion

export { default as nativeSelectClasses } from './nativeSelectClasses';
```